content/en/docs/concepts/configuration/secret.md

```diff
@@ -141,9 +178,13 @@ Each Secret you want to use needs to be referred to in `.spec.volumes`.
 If there are multiple containers in the Pod, then each container needs its
 own `volumeMounts` block, but only one `.spec.volumes` is needed per Secret.
 
-You can package many files into one secret, or use many secrets, whichever is convenient.
+{{< note >}}
+Secret volume sources are validated to ensure that the specified object
+reference actually points to an object of type Secret. Therefore, a secret
+needs to be created before any Pods that depend on it.
+{{< /note >}}
 
-#### Projection of Secret keys to specific paths
+##### Projection of Secret keys to specific paths
```

Comment:
I don't think we are encouraged to use FIVE '#' as subsections. If we do, we have a problem with content organization.

Reply:
I agree. When the content gets to heading level 5, the readability of the content decreases. That said, I need to look at the rendered page again.

Reply:
> we have a problem with content organization

This, I think. However I think this addition helps get this change mergeable. We can track further work to improve the updated page.